integrations/oci/tls-certificates/src/test/java/io/helidon/integrations/oci/tls/certificates/TestOciCertificatesDownloader.java

```diff
@@ -0,0 +1,88 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.integrations.oci.tls.certificates;
+
+import java.io.InputStream;
+import java.security.cert.Certificate;
+import java.util.Objects;
+import java.util.concurrent.TimeUnit;
+
+import io.helidon.common.Weight;
+import io.helidon.common.Weighted;
+import io.helidon.integrations.oci.tls.certificates.spi.OciCertificatesDownloader;
+
+import jakarta.inject.Singleton;
+
+@Singleton
+@Weight(Weighted.DEFAULT_WEIGHT + 1)
+class TestOciCertificatesDownloader extends DefaultOciCertificatesDownloader {
+    static String version = "1";
+
+    static int callCount_loadCertificates;
+    static int callCount_loadCACertificate;
+
+    void version(String version) {
+        TestOciCertificatesDownloader.version = version;
+    }
+
+    @Override
+    public Certificates loadCertificates(String certOcid) {
+        callCount_loadCertificates++;
+
+        try {
+            if (OciTestUtils.ociRealUsage()) {
+                return super.loadCertificates(certOcid);
+            } else {
+                TimeUnit.MILLISECONDS.sleep(1); // make sure metrics timestamp changes
+                Objects.requireNonNull(certOcid);
+                try (InputStream certIs =
+                        TestOciCertificatesDownloader.class.getClassLoader().getResourceAsStream("test-keys/serverCert.pem")) {
+                    return OciCertificatesDownloader.create(version, new Certificate[] {toCertificate(certIs)});
+                } catch (Exception e) {
+                    throw new RuntimeException(e);
+                }
+            }
+        } catch (Exception e) {
+            System.getLogger(getClass().getName()).log(System.Logger.Level.ERROR, e.getMessage(), e);
+            throw new RuntimeException(e);
+        }
+    }
+
+    @Override
+    public Certificate loadCACertificate(String caCertOcid) {
+        callCount_loadCACertificate++;
+
+        try {
+            if (OciTestUtils.ociRealUsage()) {
+                return super.loadCACertificate(caCertOcid);
+            } else {
+                TimeUnit.MILLISECONDS.sleep(1); // make sure metrics timestamp changes
+                Objects.requireNonNull(caCertOcid);
+                try (InputStream caCertIs =
+                        TestOciCertificatesDownloader.class.getClassLoader().getResourceAsStream("test-keys/ca.pem")) {
+                    return toCertificate(caCertIs);
+                } catch (Exception e) {
+                    throw new RuntimeException(e);
+                }
+            }
+        } catch (Exception e) {
```

Comment:
Why would you do this in a test class? Just trow teh exception from the the method. 
Logging an error and throwing an exception is suspicious in any code...

Reply:
I needed to catch the Interrupt exception from the `sleep()` call.  Changed it slightly.